client/src/App.js

```diff
@@ -8,11 +8,14 @@ import Search from "./components/Search/Search";
 import Navbar from "./components/Navbar/Navbar";
 
 function App() {
+  // TODO: use Paper/Cards for interface
```

Comment:
What does this mean?

Reply:
I was thinking we could `Card`s for each search result and put them in a `Paper` component, similar to how the meetup page is done right now